lib/tasks/cleanup_helper.rb

```diff
@@ -0,0 +1,72 @@
+class CleanupHelper
```

Comment:
I'm wondering about layout. Should this be a service perhaps? I'm not sure I like including something from `lib/tasks` in a migration.

Reply:
A service makes me think it's talking to something else. `app/helpers/` seems like a decent spot perhaps

Reply:
Don't helpers have a specific meaning within Rails? There's the method `helper :helper` which I think loads it, but I'll admit I'm a bit fuzzy on the details now.

Reply:
Hm yeah I think you're right, looks like this module will get loaded in. I'll look if the services dir makes sense